### PR TITLE
Quick fix scroll tracking events for GA campaigns and cosmetic changes

### PR DESCRIPTION
**Pull request by selfthinker**
This is a continuation of closed #817.
[Trello card](https://trello.com/c/ZRqYEHab/57-suppress-duplicate-scroll-tracking-events)

## Description

This PR contains five sets of changes:

* Cosmetic changes: Remove duplicate function names.
* Handling path comparison better.
* Reducing the scroll delay. By reducing the timeout delay the the scroll tracking is fired more frequently and making it hard to miss out on faster scrolling users. This at the moment works and deemed experimental. 
* To reduce the effect the reduction of the scroll delay has for users, this removes most paths which we don't need to track anymore.
* Remove all code related to percent scroll tracking as that proved not to be useful.

## Changes to previous PR

1. Renamed `concatPath` to `normalisePath` as suggested by @dsingleton.
2. @nickcolley looked at potential performance issues regarding the reduction of the scroll delay to 10 ms and found it's probably not that bad. And we initially said we will just go with it and look into potential issues later.
3. To reduce those potential issues we removed 16 out of the previous 20 paths this script would run on.
4. Remove code related to percent scroll tracking and changed tests accordingly.

@nickcolley later mentioned that a much better fix would be to use throttling instead of a timeout. As this task has been going around since 22 July and has a deadline (8 Aug) that will be properly fixed in a subsequent PR.

### CODE REVIEW

**Comment by nickcolley** [comment]
Just to clarify from trello

"I had a think about how we can improve the reliability of the events fired.

Currently we're debouncing events on scroll, this means that when a new scroll event fires it will resets the timer each time until the user stops scrolling.

We could consider throttling instead which would mean there is always consistent events firing at the interval we expect.

This should allow us to keep a respectable delay and make sure that the events fire across the page.

See http://drupalmotion.com/article/debounce-and-throttle-visual-explanation for an example of debouncing and throttling mouseevents."

In summary it seems the underlying problem was in debouncing events rather than the interval in which they were debounced.

https://trello.com/c/ZRqYEHab/57-review-time-delay-for-header-scroll-tracking-events#comment-57a1ba1fac54b5272731748b

**Comment by selfthinker** [comment]
Sorry, I tried to fix the failing test, will look at it again tomorrow. I'd be grateful If someone could give me a hint.

**Comment by tvararu** [comment]
It does look like https://github.com/alphagov/static/pull/819/files#diff-5a474506625a3d56d10f933b501c0d4cR82 is better suited to a throttle than a debounce pattern.

That being said, with the impending deadline, I think we should ship this and revise it at a later date.

LGTM. 👍

**Comment by selfthinker** [comment]
FYI, there won't be any subsequent PR as implementing throttling did not solve the original issue. I tested the implementation (34943b90f7655b75664a2e43a15a1cb950cab4cc) and found it doesn't log the scroll tracking reliably over a delay of a meagre 48ms.
We should rather keep on questioning if the scroll tracking makes sense in general and answers the right questions.
